tools/crm_resource_runtime.c

```diff
@@ -1798,9 +1798,17 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
     if (services_action_sync(op)) {
         exit_code = op->rc;
 
+        /* Lookup exit code based on rc for LSB resources */
+        if (( pcmk__str_eq(rsc_class, PCMK_RESOURCE_CLASS_LSB, pcmk__str_casei) ||
+              (pcmk__str_eq(rsc_class, PCMK_RESOURCE_CLASS_SERVICE, pcmk__str_casei) &&
+              pcmk__str_eq(resources_find_service_class(rsc_type), PCMK_RESOURCE_CLASS_LSB, pcmk__str_casei)) ) &&
+              pcmk__str_eq(rsc_action, "force-check", pcmk__str_casei)) {
```

Comment:
Check the force-check condition first, so we don't have to search the filesystem for any other options

Alternatively, if rsc_class is SERVICE, set rsc_class to resources_find... at the beginning of the function and we can ignore SERVICE thereafter